test/mocks/scs-function-name/animals-same-function-name.yaml

```diff
@@ -0,0 +1,38 @@
+components:
```

Comment:
We already had a file like this for dynamic topics so I made another one here for static ones that ensure uses Function instead of Consumer.